pkg/renderer/readme.go

```diff
@@ -40,6 +40,7 @@ the login page, and log in with username "admin" and the password found in the
 	kubectlInstructionTemplate = `
   - Deploy Central
     - Run central/scripts/setup.sh
+    - Run {{.K8sConfig.Command}} apply -f helm/chart/crds/config.stackrox.io_securitypolicies.yaml
```

Comment:
I was choosing to put it in the readme over the script because it matches Helm's attitude "delegate most CRD management to the user."  Helm creates the CRD but then refuses to touch it after the initial creation.  Should we change the command to `kubectl create` instead of `kubectl apply`?   That would then fail if the CRD already existed, leaving it to the customer to figure out how to best ugprade the CRD.